dragon/docs.rb

```diff
@@ -267,6 +270,37 @@ def __docs_to_html__ string
 S
     html_content_end_to_html_end = <<-S
     </div>
+    <script src="prism.js" data-manual></script>
+    <script>
+      document.addEventListener("DOMContentLoaded", function() {
+              var lazyloadElements = document.querySelectorAll("code");
```

Comment:
You probably should only target `pre > code`.... since all the inline code elements are also formatted right now...

Reply:
Done.